tests/test_aks.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import kubernetes
+from ocw.lib.provider import Provider
+from ocw.lib.aks import AKS
+from webui.PCWConfig import PCWConfig
+from tests.kubernetes import MockedSubprocessReturn, MockedKubernetesClient, MockedKubernetesConfig, MockedKubernetesJob
+
+
+class MockedAKSCluster():
+    def __init__(self, name):
+        self.name = name
+
+
+@pytest.fixture
+def aks_patch(monkeypatch):
+    monkeypatch.setattr(Provider, 'read_auth_json',
+                        lambda *args, **kwargs: {'client_id': 'key', 'client_secret': 'secret',
+                                                 'subscription_id': 'subscription', 'tenant_id': 'tenant',
+                                                 'resource_group_name': 'group'})
+    monkeypatch.setattr(AKS, "get_clusters", lambda *args, **kwargs: [MockedAKSCluster('cluster1')])
+
+    return AKS('fake')
+
+
+def test_kubectl_client(aks_patch, monkeypatch):
+    monkeypatch.setattr(Provider, 'cmd_exec', lambda *args, **kwargs: MockedSubprocessReturn(0))
+    monkeypatch.setattr(kubernetes, 'config', MockedKubernetesConfig())
+    mocked_client1 = MockedKubernetesClient(1)
+    monkeypatch.setattr(kubernetes, 'client', mocked_client1)
+
+    assert aks_patch.kubectl_client("group", "cluster") == mocked_client1
+
+    # Check that the client is reused
+    mocked_client2 = MockedKubernetesClient(1)
+    monkeypatch.setattr(kubernetes, 'client', mocked_client2)
+    assert aks_patch.kubectl_client("group", "cluster") == mocked_client1
+
+
+def test_cleanup_k8s_jobs(aks_patch, monkeypatch):
+    monkeypatch.setattr(kubernetes, 'config', MockedKubernetesConfig())
```

Comment:
this line presented in both tests hence you can move it into `aks_patch`

Reply:
done